.env.sample

```diff
@@ -1,23 +1,38 @@
+# NODE
 NODE_ENV=
+
+# DATABASE
+DATABASE_PORT=
+MYSQL_DATABASE=
+MYSQL_ROOT_PASSWORD=
+DATABASE_URL=
+
+# API
+API_PORT=
+API_URL=
+SESSION_SECRET
+JWT_SIGNING_KEY=
+JWT_VALIDATING_KEY=
+MOCK_ACCOUNT_PASSWORD
```

Comment:
`MOCK_ACCOUNT_PASSWORD` is not specific to the API it's used in E2E and API

Reply:
Have corrected